### ISSUE TITLE
Filtering by 'Date of record input' doesn't work

### ISSUE THREAD

**Issue by DavidHepper** [BiologicalRecordsCentre/iRecord#1602]
In Create a filter > When > Date field: 'Date of record input' and with Year set to 'equals' e.g. 2022, the grid shows records recorded in that year, not input in that year.

**Comment by kitenetter** [comment]
Agreed - the "Year" filter only filters on date of field record, and choosing any of the other "Date field" options has no effect.

**Comment by DavidHepper** [comment]
OK but that's not what the form is currently offering to the user. Either (preferably) allow the same level of control of any of the date fields or grey out the 'Year' field whenever a 'Date field' other than 'Field record date' is selected.

**Comment by kitenetter** [comment]
Yes, I agree with you @DavidHepper ! Was just trying to point out that none of the options work other than field date, didn't mean to suggest that is the intended behaviour! Ideally all the options should be made to work, but if that isn't possible then greying out the year option would be an alternative.

**Comment by DavidHepper** [comment]
Sorry! I took it the wrong way. (Didn't notice the Bug label already attached.)

**Comment by johnvanbreda** [comment]
Duplicate of #1774 - fix will be deployed shortly.
